images/base/entrypoint

```diff
@@ -19,12 +19,15 @@ set -o nounset
 set -o pipefail
 
 configure_proxy() {
+  # Don't use proxy for the local network
+  LOCAL_NETWORK=$(ip route list dev eth0 proto kernel | awk '{ print $1 }')
```

Comment:
minor q: is `ip route list` the same as `ip route show`? so far can't find manpages for list :^)

Reply:
I'm more used to `ip route show` too, however the output of the ip help doesn't mention show
```
ip route help
Usage: ip route { list | flush } SELECTOR
       ip route save SELECTOR
       ip route restore
       ip route showdump
       ip route get [ ROUTE_GET_FLAGS ] ADDRESS
```